site/src/components/pages/user/TabPanelContentsWithSchemas.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box } from "@mui/material";
+import { useState, VoidFunctionComponent } from "react";
+import { Button } from "../../Button";
+import { EntityType } from "../../../lib/api/model/entityType.model";
+import { SerializedUser } from "../../../lib/api/model/user.model";
+import { ListViewCard } from "./ListViewCard";
+import { CreateSchemaModal } from "../../Modal/CreateSchemaModal";
+import { useUserStatus } from "./useUserStatus";
+import { Placeholder } from "./Placeholder";
+import { BrowseHubButton, CreateSchemaButton } from "./PlaceholderButtons";
+
+export interface TabPanelContentsWithSchemasProps {
+  entityTypes: EntityType[];
+  user: SerializedUser;
+}
+
+export const TabPanelContentsWithSchemas: VoidFunctionComponent<
+  TabPanelContentsWithSchemasProps
+> = ({ user, entityTypes }) => {
+  const [schemaModalOpen, setSchemaModalOpen] = useState(false);
+  const userStatus = useUserStatus(user);
+  const createSchemaModal = (
+    <CreateSchemaModal
+      open={schemaModalOpen}
+      onClose={() => setSchemaModalOpen(false)}
+    />
+  );
+
+  if (!entityTypes.length) {
+    if (userStatus === "loading") {
+      return null;
+    }
+
+    return userStatus === "current" ? (
+      <>
+        <Placeholder
+          header="You haven’t created any schemas yet"
+          tip="Start building to see your creations show up here."
+          actions={
+            <CreateSchemaButton onClick={() => setSchemaModalOpen(true)} />
+          }
+        />
+        {createSchemaModal}
+      </>
+    ) : (
+      <Placeholder
+        header={`@${user.shortname} hasn’t published any schemas yet`}
+        tip="You can browse existing schemas on the Block Hub."
+        actions={<BrowseHubButton />}
```

Comment:
I wonder if we should add the `Browse Block Hub` button for schemas. We currently don't display schemas in the block hub cc @nonparibus